lms/services/d2l_api/_basic.py

```diff
@@ -89,4 +97,7 @@ def api_url(self, path, product="lp"):
 
             https://docs.valence.desire2learn.com/basic/conventions.html#term-D2LPRODUCT
         """
-        return f"https://{self.lms_host}/d2l/api/{product}/{API_VERSION}{path}"
+        # Not using a default value here.
+        # If we use a new product we must include the version requirements explicitly  in `API_VERSIONS'
+        api_version = API_VERSIONS[product]
```

Comment:
I rather see a KeyError in sentry than either calling the endpoint with some random version (like before this PR) or even worse calling `/version/None/...`